resources/css/_settingsManager.css

```diff
@@ -23,8 +23,8 @@ QLabel#settingsLabel {
     font-weight: bold;
 }
 QSpinBox,
-QLabel#downloadDirPath {
-    font: bold 12pt "Cantarell";
```

Comment:
Is that normal that the font has disappeared here?

Reply:
Everywhere else, there was no font attribute. So I think it's globally added already, no need to add again